code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -101,47 +101,67 @@
 
 	var/datum/port/input/message
 	var/datum/port/input/send_message_signal
+	var/datum/port/input/show_charge_meter
 
 	var/datum/port/output/user_port
 
 	var/datum/weakref/user
 
+	var/obj/item/organ/internal/cyberimp/bci/bci
```

Comment:
Set this to null in Destroy

Reply:
Why? Other components don't do this.

Reply:
Oh right this gets removed in unregister_shell, ok its fine then